src/main/java/uk/gov/pay/api/service/SearchRefundsService.java

```diff
@@ -0,0 +1,84 @@
+package uk.gov.pay.api.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.apache.commons.lang3.StringUtils;
+import uk.gov.pay.api.app.config.PublicApiConfig;
+import uk.gov.pay.api.auth.Account;
+import uk.gov.pay.api.exception.ValidationException;
+import uk.gov.pay.api.model.search.card.SearchRefunds;
+
+import javax.inject.Inject;
+import javax.ws.rs.client.Client;
+import javax.ws.rs.core.Response;
+import java.util.LinkedHashMap;
+import java.util.LinkedList;
+import java.util.List;
+import java.util.Map;
+
+import static org.apache.commons.lang3.StringUtils.join;
+import static org.eclipse.jetty.util.StringUtil.isNotBlank;
+import static uk.gov.pay.api.model.PaymentError.Code.SEARCH_PAYMENTS_VALIDATION_ERROR;
+import static uk.gov.pay.api.model.PaymentError.aPaymentError;
+
+public class SearchRefundsService {
+    private static final String PAGE = "page";
+    private static final String DISPLAY_SIZE = "display_size";
+    private final ConnectorUriGenerator uriGenerator;
+    private final Client client;
+    private final ObjectMapper objectMapper;
+    private final PublicApiConfig configuration;
+
+    @Inject
+    public SearchRefundsService(Client client,
+                                PublicApiConfig configuration,
+                                ConnectorUriGenerator uriGenerator,
+                                ObjectMapper objectMapper) {
+
+        this.client = client;
+        this.configuration = configuration;
+        this.uriGenerator = uriGenerator;
+        this.objectMapper = objectMapper;
+    }
+
+    public Response getAllRefunds(Account account, String page, String displaySize) {
```

Comment:
we might want to have a class (maybe an inner one, maybe not) for RefundSearchParams. Right now we only have page and display size, but this will grow and we don't want this method to become a monster with 100 parameters. This should also avoid you having to stick stuff in a map, which is always a bit ...sketchy